src/lib/location.ts

```diff
@@ -107,7 +107,14 @@ export async function resolveLocation(
     };
   } else {
     const { address } = zipOrAddress;
-    const response = await geocoder.geocode(address, ['census2024']);
+    let response;
+
+    try {
+      response = await geocoder.geocode(address, ['census2024']);
+    } catch (error) {
+      return null;
```

Comment:
Let's log the error here with `console.error`; it currently doesn't go anywhere except Cloud Run logs but it could be useful to have.